### DISCUSSION TITLE
Metrics for Strimzi Operator

### DISCUSSION THREAD

**Discussion by anandswaminathan** [strimzi/strimzi-kafka-operator#6190] [Q&A] [answered]
Hello,

I have Strimzi Operator deployed that is managing KafkaConnect. I could find documentation and setup metrics for Kafka Connect to publish to Prometheus.

But I am unable to get metrics from within the operator. Where can I find instructions to setup metrics for the operator. I am interested in metrics like: strimzi_reconciliations_successful_total, strimzi_reconciliations_failed_total.

If I exec into the operator pod, and execute 
```
 curl http://localhost:9404/metrics
```
I get `curl: (7) Failed to connect to localhost port 9404: Connection refused`

**Comment by sknot-rh** [comment, marked as answer]
Hello. Cluster operator metrics are available at `health` port. By default, it is set to 8080.

**Reply by anandswaminathan** [reply]
Thanks that worked.
